### PR TITLE
Add DXF server export params NO_MTEXT and FORCE_2D

### PR DESCRIPTION
**Pull request by m-kuhn**
# Changelog

QGIS server now supports the new parameters `NO_MTEXT` and `FORCE_2D` to control text and line symbology for generated DXF files.

---------

# PR description

Adds missing parameters to GetDxf request

### CODE REVIEW

File: src/server/services/wms/qgswmsparameters.h [hunk 1]
```diff
@@ -1309,6 +1311,22 @@ namespace QgsWms
...
+       */
+      bool noMText() const;
+
+      /**
+       * \returns true if the FORCE_2D parameter is set and the DXF should
+       * be produced in 2D.
+       *
+       * \since QGIS 3.12
+       */
+      bool force2D() const;
```
Comment (lbartoletti) [inline src/server/services/wms/qgswmsparameters.h]: isForce2D? is2D?

Comment (m-kuhn) [reply src/server/services/wms/qgswmsparameters.h]: What's wrong with the current one?

Comment (m-kuhn) [reply src/server/services/wms/qgswmsparameters.h]: Ah, it could be misread as an imperative command instead of a getter?

Comment (lbartoletti) [reply src/server/services/wms/qgswmsparameters.h]: Yes :)

**Review by pblottiere — APPROVED** [review]

**Review by lbartoletti — APPROVED** [review]

**Comment by m-kuhn** [comment]
Thanks for the reviews and comments and @pblottiere and @lbartoletti

**Pull request merged**